doc/book/config-post-processors.md

```diff
@@ -0,0 +1,90 @@
+# Config post processors
+
+The `ConfigAggregator` can apply post processors to the merged configuration
+by aggregating "config processors" passed to its
+constructor.  Each processor should be a callable, which consumes the merged configuration
+as parameter and returning a configuration array.
+
+```php
+$providers = [
+    function () {
+        return ['foo' => 'bar'];
+    },
+    new PhpFileProvider('*.global.php'),
+];
+
+$processors = [
+    function (array $config) {
+        return $config + ['post-processed' => true];
+    },
+];
+
+$aggregator = new ConfigAggregator($providers, null, $processors);
+var_dump($aggregator->getMergedConfig());
+```
+
+If the processor is a class name, the aggregator automatically instantiates it
+before invoking it; as such, any class name you use as a config provider _must_
+also define `__invoke()`, and that method _must_ return an array and _may_ consume
+the merged configuration as a parameter.
+
+This can be used to mimic the Symfony ParameterBag system: you can specify a config
+processor which consumes the merged configuration and resolves the parameters which
+might be used in your configuration.
+
+Output from the example:
+
+```php
+array(2) {
+  'foo' =>
+  string(3) "bar"
+  'post-processed' =>
+  bool(true)
+}
+```
+
+## Available config post processors
+
+### Symfony ParameterBag Post Processor
```

Comment:
This isn't an _available_ (as in shipped) post processor; it's an _example_ of how to provide such functionality _via_ a processor, isn't it?

I'm going to operate on that assumption, and push a commit for you to review.

Reply:
Thats fine for me. I had a pre-shipped version but got hilarious issues with phpunit and therefore cut it off.